code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -346,7 +346,7 @@
 	required_reagents = list("stable_plasma" = 1, "radium" = 1, "phosphorus" = 1)
 
 /datum/chemical_reaction/pyrosium/on_reaction(datum/reagents/holder, created_volume)
-	holder.chem_temp = 20 // also cools the fuck down
+	holder.chem_temp += 500 // also heats the fuck up AGAIN
```

Comment:
This was here because these chemicals were meant to be like timers so they both would start at the same temp when made 20 then would either rise or fall in the case of cryostylane if there is oxygen to react with. Grenades vanish after a while though so its only useful in say watertanks for a reaction.